ts/models/message.ts

```diff
@@ -361,7 +375,7 @@ export class MessageModel extends Backbone.Model<MessageAttributes> {
   public getPropsForGroupUpdateMessage(): PropsForGroupUpdate | null {
     const groupUpdate = this.getGroupUpdateAsArray();
 
-    if (!groupUpdate || _.isEmpty(groupUpdate)) {
+    if (!groupUpdate || isEmpty(groupUpdate)) {
       return null;
     }
```

Comment:
this might be a change of mine,
but I would think the isEmpty check for null and undefined already